src/Tizen.NUI/src/public/BaseComponents/ViewEnum.cs

```diff
@@ -163,6 +163,32 @@ public enum FocusDirection
             CounterClockwise,
         }
 
+        /// <summary>
+        /// Describes offscreen rendering types.
+        /// View with this property enabled renders at offscreen buffer, with all its children.
+        /// The property expects to reduce many repetitive render calls.
+        /// </summary>
+        /// <since_tizen> 12 </since_tizen>
```

Comment:
```suggestion
        [EditorBrowsable(EditorBrowsableState.Never)]
```

Please also make this enum as Never.